### PR TITLE
Add references to the format list

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**

### CODE REVIEW

File: draft-ietf-scitt-architecture.md [hunk 1]
```diff
@@ -74,7 +74,25 @@ normative:
     date: 2022-07-22
   DID-WEB:
     target: https://w3c-ccg.github.io/did-method-web/
-    title: did:web Decentralized Identifiers Method Spec
+    title:  did:web Decentralized Identifiers Method Spec
+  SPDX:
+    target: https://spdx.dev/use/specifications/
+    title:  SPDX Specification
+  SWID:
+    target: https://csrc.nist.gov/projects/software-identification-swid/
```
Comment (aj-stein-nist) [inline draft-ietf-scitt-architecture.md:82]: I will venture a guess that you can be more specific if you point to either.

The NIST specifications:

```suggestion
    target: https://csrc.nist.gov/Projects/Software-Identification-SWID/guidelines
```

Or the referenced ISO specifications, which is what NIST recommends you do on the changed link above.

> The SWID Tag format, defined by the International Organization for Standardization (ISO) and the International Electrotechnical Commission (IEC) standard ISO/IEC 19770-2, is a structured metadata format for describing a software product. NIST recommends use of the latest version of this standard, [ISO/IEC 19770-2:2015](https://webstore.ansi.org/RecordDetail.aspx?sku=ISO%2fIEC+19770-2%3a2015).

```suggestion
    target: https://www.iso.org/standard/65666.html
```

File: draft-ietf-scitt-architecture.md [hunk 2]
```diff
@@ -74,7 +74,25 @@ normative:
     date: 2022-07-22
   DID-WEB:
     target: https://w3c-ccg.github.io/did-method-web/
-    title: did:web Decentralized Identifiers Method Spec
+    title:  did:web Decentralized Identifiers Method Spec
```
Comment (aj-stein-nist) [inline draft-ietf-scitt-architecture.md:77]: Is there a reason for the double space?

```suggestion
    title: did:web Decentralized Identifiers Method Spec
```

**Review by aj-stein-nist — COMMENTED** [review]
Minor nits around spacing and a URL update (pretending I understand my employer).

**Review by SteveLasker — APPROVED** [review]
LGTM

**Comment by SteveLasker** [comment]
To minimize the merge conflicts, I'll merge after #118

**Comment by yogeshbdeshpande** [comment]
lets merge for 118.

**Review by aj-stein-nist — APPROVED** [review]

**Comment by yogeshbdeshpande** [comment]
ok, please merge your PR and then I will Rebase my (this) PR

**Review by hannestschofenig — APPROVED** [review]
Thanks for adding the references.

I am not sure whether you can indeed add them in this style but you can try. I would just use this style:

SLSA {{SLSA}}

**Pull request merged**